orm/models.py

```diff
@@ -170,8 +172,8 @@ def limit(self, rows_to_return: int):
             model_cls=self.model_cls,
             filter_clauses=self.filter_clauses,
             select_related=self._select_related,
+            limit_count=rows_to_return
```

Comment:
I'd suggest we call the argument `limit` or `limit_count` or something.

Reply:
Right makes sense, I should have kept them consistent. I've updated these to be the same (using limit_count)